netmiko/extreme/extreme_ssh.py

```diff
@@ -1,13 +1,123 @@
 """Extreme support."""
 from __future__ import unicode_literals
+import re
+import time
 from netmiko.cisco_base_connection import CiscoSSHConnection
 
 
+QUESTIONS = {
+    'save': 'Do you want to save configuration to',
+}
+
+
 class ExtremeSSH(CiscoSSHConnection):
     """Extreme support."""
+
     def session_preparation(self):
-        """Extreme requires enable mode to disable paging."""
+        """
+        Extreme requires enable mode to disable paging.
+        """
         self._test_channel_read()
-        self.enable()
         self.set_base_prompt()
         self.disable_paging(command="disable clipaging\n")
```

Comment:
I have an earlier note that `enable` was required to disable paging for Extreme. Is this false? Leaving the `enable` call shouldn't hurt anything (if we are unsure).